decidim-admin/app/packs/src/decidim/admin/choose_language.js

```diff
@@ -1,12 +1,14 @@
 /* eslint-disable no-invalid-this */
+/* eslint-disable require-jsdoc */
 
-$(() => {
-  $("select.language-change").change(function () {
-    let $select = $(this);
-    let targetTabPaneSelector = $select.val();
-    let $tabsContent = $select.parent().parent().siblings();
+export default function initLanguageChangeSelect(elements) {
+  elements.forEach((select) => {
+    select.onchange = function () {
```

Comment:
Could you use `select.addEventListener("click")` instead? Overriding the value for `select.onchange` can have some side effects. Also can we use arrow function here as you already have access to the correct `select` element in this context which you can use instead of `this` in the event handler.

```suggestion
    select.onchange = () => {
```

With the current approach with the exported method used in the two places, we shouldn't have multiple listeners bound.

Reply:
Good point! :+1: